### DISCUSSION TITLE
Issues with Tailwind CSS Styles Not Rendering Correctly in FilamentPHP Custom Widgets

### DISCUSSION THREAD

**Discussion by adiartawibawa** [filamentphp/filament#13868] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

I am working with FilamentPHP and attempting to use a custom widget with Tailwind CSS. I have the following Livewire component:

```php
namespace App\Livewire\Widgets\Users;

use Filament\Widgets\Widget;
use Illuminate\Contracts\View\View;

class TotalUserWidget extends Widget
{
    protected static string $view = 'livewire.widgets.users.total-user-widget';
}
```

The corresponding view is as follows:

```html
<div class="bg-gradient-to-r from-indigo-500 via-purple-500 to-pink-500 rounded-lg shadow-lg p-8">
    <h1 class="text-4xl font-bold text-white mb-4">
        Tailwind CSS
    </h1>
    <p class="text-lg text-primary-500 mb-8">
        A utility-first CSS framework for rapidly building custom designs.
    </p>
    <a href="#" class="bg-white text-primary-500 font-bold py-2 px-4 rounded">
        Get started
    </a>
</div>
```

I placed this component in the Filament admin panel using the following code:

```php
->widgets([
    TotalUserWidget::class,
])
```

Unfortunately, when I run the code, the styles do not render as expected. However, when I run the code outside of the admin panel, everything works correctly.
what i get:
![image](https://github.com/user-attachments/assets/2b6b90ac-87aa-4190-a428-7c0f49a02abd)

what i want:
![image](https://github.com/user-attachments/assets/53aa70e2-4828-4512-a20e-0b2c0e532ba3)

Could you please help me understand why the styles are not being applied correctly within the Filament admin panel and how I might resolve this issue?

**Comment by leandrocfe** [comment, marked as answer]
You need to create a custom theme

https://filamentphp.com/docs/3.x/panels/themes/#creating-a-custom-theme
